docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+services:
+  db:
+    image: mysql:5.7.22
+    command: mysqld --character-set-server=utf8 --collation-server=utf8_unicode_ci
+    environment:
+      MYSQL_DATABASE: yaronet
+      MYSQL_USER: root
+      MYSQL_ROOT_PASSWORD: password
+    ports:
+      - '3306:3306'
+    hostname: db
+    volumes:
+      - ./tmp/db:/var/lib/mysql
+  web:
+    build: .
+    command: /bin/sh -c /yaronet/setup/docker/entrypoint.sh
+    volumes:
+      - .:/yaronet
+    environment:
+      APP_HOME: /yaronet
+      DB_HOST: 'db'
+      DB_PORT: '3306'
```

Comment:
Seems this configuration is slightly different from the default one used in Vagrant-based install. It means user will have to change default SQL connection string from `mysqli://yaronet@localhost/yaronet?charset=utf8mb4` to `mysqli://root:password@db/yaronet?charset=utf8mb4` ; I would suggest keeping the same configuration everywhere so that using defaults works everywhere.

Reply:
Ok, let me change the config to match default values.

Reply:
I realize that db host cannot be localhost.
And I realize that mysql client is not installed on php container, and it would be better to keep it this way.

I suggest:
- keep php script, it is not harmful
- add a comment in the doc to adjust the hostname when configuring the site

Reply:
Changed user from root to yaronet.
Updated doc with clear config to input.